mis_builder/tests/test_mis_builder.py

```diff
@@ -65,6 +45,7 @@ def test_fetch_query(self):
                                                   'mis_report_instance_'
                                                   'period_test'),
                             'style': None,
+                            'prefix': False,
```

Comment:
In this file, please CamelCase the class name (while you are at it), and at line 53, write ```'val_r': u'\u202f0\u00a0',``` to fix the test.